polygeist/lib/polygeist/CMakeLists.txt

```diff
@@ -11,5 +11,10 @@ MLIRPolygeistOpsIncGen
 LINK_LIBS PUBLIC
 MLIRIR
 MLIRMemRefDialect
+MLIRAffineDialect
```

Comment:
Can you pls put this list in alphabetical order.

Reply:
Done